eq-author-api/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
       - DYNAMO_COMMENTS_TABLE_NAME=dev-author-comments
       - NODE_ENV=development
       - RUNNER_SESSION_URL=http://localhost:5000/session?token=
-      - PUBLISHER_URL=http://localhost:9000/publish/
+      - PUBLISHER_URL=http://host.docker.internal:9000/publish/
```

Comment:
I think this will break "View survey" for people who are not running runner with docker-compose.

Reply:
Not sure if it has any other implications?

Reply:
Left as docker-compose version